site-manage/site-manage-group-section-role-helper/tool/src/java/org/sakaiproject/site/tool/helper/managegroupsectionrole/impl/SiteManageGroupSectionRoleHandler.java

```diff
@@ -340,22 +340,19 @@ public String getRosterLabel( String rosterID )
         String label = rosterID;
         try
         {
-            if (cms != null)
```

Comment:
It's misleading, cms is always defined. I think this changed a while ago in Sakai and I never caught up.